mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -109,6 +110,13 @@ public class JobExecutionDaoImpl implements JobExecutionDao {
   public static final String TENANT_NAME = "tenantName";
   public static final String TRUE = "true";
   public static final String DB_TABLE_NAME_FIELD = "tableName";
+  public static final String SELECT_ID_WHERE_IS_DELETED_TRUE = "SELECT id FROM %s WHERE is_deleted = true";
+  public static final String DELETE_QUERY = "DELETE from %s where %s IN ('%s')";
+  public static final String JOB_MONITORING = "job_monitoring";
+  public static final String JOB_EXECUTION_SOURCE_CHUNKS = "job_execution_source_chunks";
+  public static final String JOURNAL_RECORDS = "journal_records";
+  public static final String JOB_EXECUTION_ID = "job_execution_id";
+  public static final String JOBEXECUTIONID = "jobexecutionid";
```

Comment:
Its needed to be refactored in some way to avoid to identical by meaning constant names: JOB_EXECUTION_ID, JOBEXECUTIONID

Reply:
Refactored to use JOB_EXECUTION_ID and JOB_EXECUTION_ID_OLD_FORMAT constants